### ISSUE TITLE
imported dataclasses are not checked

### ISSUE THREAD

**Issue by socketpair** [python/mypy#10497]
```python
from dataclasses import dataclass

@dataclass(frozen=True)
class SystemVersion:
    major: int


x = SystemVersion()
```

Errors:
```
zzzz.py:11: error: Too few arguments for "SystemVersion"  [call-arg]
```

It's OK. But if the class is imported from outside of the code I check (i.e. from third party library), mypy does not generate the same error.

**Comment by JelleZijlstra** [comment]
Do you have the `--ignore-missing-imports` flag on? What does `reveal_type(SystemVersion)` show?

**Comment by socketpair** [comment]
No, I did not enable this flag.
How to use `reveal_type(SystemVersion)` ?

**Comment by socketpair** [comment]
@JelleZijlstra 
`note: Revealed type is 'Any'`

**Comment by socketpair** [comment]
Sorry. I HAVE `ignore_missing_imports = True`.

**Comment by socketpair** [comment]
But seem this import is not missing. Will investigate

**Comment by socketpair** [comment]
```
system_backend/zzzz.py:3: error: Skipping analyzing 'ideco_version': found module but no type hints or library stubs  [import]
system_backend/zzzz.py:3: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
system_backend/zzzz.py:11: note: Revealed type is 'Any'
```

**Comment by socketpair** [comment]
But it HAS type hints. It's my own library fully covered by type hints.

**Comment by socketpair** [comment]
I have mypy.ini:
```
[mypy]
show_error_codes = True
show_error_context = True
```

And no command-line options in my CI. So what could be wrong ?

**Comment by socketpair** [comment]
I did not have `py.typed` in roots of my own packages. SUPRISE. :(
